overlay/qrtr/rmtfs.nix

```diff
@@ -17,6 +17,9 @@ stdenv.mkDerivation {
     };
 
     patchPhase = ''
+        if !which qmic >/dev/null 2>/dev/null ; then
+            sed -i 's/qmic/qmicli/' Makefile
+        fi
```

Comment:
Why is this done with a conditional?

Dependencies should be pretty strongly declared. We're free, and it's acceptable, to only support the newer name.

Anything I'm missing?

Reply:
yes, you definitely dont need to run qmic as part of packaging, this distinction should probably be made better upstream. Maybe its worth carrying a patch if necessary to allow building without qmic for the purposes of packaging.